src/themes/material/templates/repository/elements/subject_tree.html

```diff
@@ -0,0 +1,8 @@
+{% for subject in subjects %}
+    <li><a href="{% url 'preprints_list_subject' subject.slug %}">{{ subject }}</a></li>
```

Comment:
So it looks like the view routed from this URL predates this PR. I had a look at it and I don't think it does what it is supposed to do:
https://github.com/BirkbeckCTP/janeway/blob/d89a6114b0ed280b63f210316937ec0dcd989ea8/src/repository/views.py#L199

It fetches the subject but does not apply any filters on the queryset.

While we are at it, is it worth renaming this to repository_list_subject to be consistent with the other views?

Reply:
It grabs the subject and then fetches its preprint_set see:

https://github.com/BirkbeckCTP/janeway/blob/d89a6114b0ed280b63f210316937ec0dcd989ea8/src/repository/views.py#L201

Reply:
:+1: